src/test/java/com/synopsys/integration/alert/component/users/web/role/RoleActionsTest.java

```diff
@@ -44,11 +45,17 @@ public void init() {
         roleAccessor = Mockito.mock(RoleAccessor.class);
         descriptorKey = createDescriptorKey("descriptorKey-test");
 
-        Mockito.when(authorizationManager.hasCreatePermission(Mockito.any(), Mockito.any())).thenReturn(true);
-        Mockito.when(authorizationManager.hasReadPermission(Mockito.any(), Mockito.any())).thenReturn(true);
-        Mockito.when(authorizationManager.hasDeletePermission(Mockito.any(), Mockito.any())).thenReturn(true);
-        Mockito.when(authorizationManager.hasExecutePermission(Mockito.any(), Mockito.any())).thenReturn(true);
-        Mockito.when(authorizationManager.hasWritePermission(Mockito.any(), Mockito.any())).thenReturn(true);
+        Mockito.when(authorizationManager.hasCreatePermission(Mockito.any(ConfigContextEnum.class), Mockito.any(DescriptorKey.class))).thenReturn(true);
```

Comment:
I added mocks for both methods here since the class being tested could call the deprecated methods. I was thinking of changing the mock to throw an exception when the deprecated methods are called. Thoughts?